builtin/providers/azurerm/resource_arm_virtual_machine.go

```diff
@@ -190,7 +208,24 @@ func resourceArmVirtualMachine() *schema.Resource {
 
 						"vhd_uri": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
```

Comment:
Ok, sounds good to me. I've opened issue #13016 for the ConflictsWith problem. I'll pull the changes out and submit a PR for that issue shortly.